### PR TITLE
testing

### PR DESCRIPTION
**Pull request by kelseyhuse30**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Company data addition
- [ ] Bug fix
- [ ] New feature

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:
* Browser:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: companies/github.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ tags: ["tech"]
 blm_statements:
   - url: https://twitter.com/natfriedman/status/1266790816174882816?s=20
     date_posted: 2020-05-30
-    type: Tweet
-    raw_text: This week has been a horrifying, sad reminder of the centuries-long pattern of systemic racism in the US. And that our criminal justice system is in dire need of reform. GitHub stands with the Black community and will not be silent on violence and injustice.
+    type Tweet
```
Comment (kelseyhuse30) [inline companies/github.md:7]: @Nyette I would expect this file to error but looks like it passed 😭

Comment (Nyette) [reply companies/github.md:7]: We may have to play around with the [indentation](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.indentation) rule

Comment (Nyette) [reply companies/github.md:7]: May I add a configuration file?

Comment (kelseyhuse30) [reply companies/github.md:7]: yep definitely!

File: companies/github.md [hunk 2]
```diff
@@ -4,12 +4,8 @@ tags: ["tech"]
...
   - url: https://twitter.com/natfriedman/status/1266790816174882816?s=20
     date_posted: 2020-05-30
-    type: Tweet
-    raw_text: This week has been a horrifying, sad reminder of the centuries-long pattern of systemic racism in the US. And that our criminal justice system is in dire need of reform. GitHub stands with the Black community and will not be silent on violence and injustice.
-  - url: https://twitter.com/github/status/1269664807977771008?s=20
-    date_posted: 2020-06-07
-    type: Tweet
-    raw_text: Breonna Taylor, George Floyd, Ahmaud Arbery, Tony McDade, and countless others deserve justice. We extend our support to Black Hubbers and Black developers. GitHub stands with the Black community in the fight against racism and injustice, and will continue to take action.
+    type Tweet
+    raw_text: -Test This week has been a horrifying, sad reminder of the centuries-long pattern of systemic racism in the US. And that our criminal justice system is in dire need of reform. GitHub stands with the Black community and will not be silent on violence and injustice.
```
[the remark this answers is not in the document]
Comment (kelseyhuse30) [reply companies/github.md:8]: @Nyette it's working!!!

[the remark this answers is not in the document]
Comment (Nyette) [reply companies/github.md:8]: I'm happy to hear that!

**Pull request closed without merging**